config/chewy.rb

```diff
@@ -0,0 +1,25 @@
+Chewy.logger = Rails.logger
+
```

Comment:
@mkcode I did not, I'm still diving into Chewy to see what it can do.

This looks great, thank you for the pointer.